load-package.cjs

```diff
@@ -0,0 +1,3 @@
+// file needed for importing package-json
+// see https://rollupjs.org/guide/en/#using-untranspiled-config-files
+module.exports = require('./package.json');
```

Comment:
basically what the comments say, this is required for loading the package.json in es modules.